src/lib/Behat/PageObject/Page.php

```diff
@@ -33,6 +37,13 @@ public function verifyIsLoaded(): void
     {
         $this->verifyRoute();
         $this->verifyElements();
+        if ('/admin/login' !== $this->route) {
```

Comment:
1) Why aren't we doing it for LoginPage?
2) If we really want to skip the LoginPage, approach should be different imho:
create new method verifyTitle (with default implementation here), and overload it in LoginPage?

Reply:
Simply because of that, that LoginPage has no title - it's just login form :)